environs/bootstrap/bootstrap.go

```diff
@@ -479,15 +511,10 @@ func bootstrapImageMetadata(
 	}
 
 	if bootstrapImageId != "" {
-		arches := availableTools.Arches()
-		if len(arches) != 1 {
-			return nil, errors.NotValidf("multiple architectures with bootstrap image")
+		if bootstrapSeries == nil {
+			return nil, errors.NotValidf("no series specified with bootstrap image")
 		}
-		allSeries := availableTools.AllSeries()
-		if len(allSeries) != 1 {
-			return nil, errors.NotValidf("multiple series with bootstrap image")
-		}
-		seriesVersion, err := series.SeriesVersion(allSeries[0])
+		seriesVersion, err := series.SeriesVersion(*bootstrapSeries)
```

Comment:
The rational is that we used to get series from tools we've loaded. Now we are getting image metadata before we get tools. So we know there is only one.
However, you are right that I have remove the error and we should definitely err if bootstrap series is nil here. Adding the check :D